tests/scenarios/test_functionality_nimbus.py

```diff
@@ -37,3 +64,32 @@ def check_url_page_loads_correctly(selenium):
         EC.visibility_of_element_located((By.CSS_SELECTOR, ".c-navigation-logo-image")),
         message="Navigation logo not found",
     )
+
+
+@then("Firefox should still accept a copied string that is sent to the search bar")
+def copy_and_paste_string_to_url_bar(selenium, navigate_using_url_bar):
+    cmd_ctrl = Keys.COMMAND if sys.platform == "darwin" else Keys.CONTROL
```

Comment:
I think I saw this above on L17, although not 100% convinced it could/should be abstracted into some helper. Also didn't check the codebase to see if we're doing OS specific keys in other tests. But just throwing it out there for future consideration.